# Chapter45
## 📌문제1

### true or false를 판단하세요.

```
(1) 비동기 함수 내부의 비동기로 동작하는 코드는 비동기 함수가 종료된 이후에 완료된다.
(2) settled 상태에서 비동기 처리가 수행 성공되면 resolve 함수를 호출하여 프로미스를 fulfilled 상태로 변경한다.
(3) 에러 처리는 catch 메서드보다 then 메서드의 두 번째 콜백 함수를 사용하는 것이 더 권장된다.
```

### 답안 작성
```
(1) 
(2) 
(3) 
```

<br>

## 📌문제2

### 다음 코드 중, catch 코드 블록에서 에러 캐치가 되지 않는 설명하세요.
```js
try {
  setTimeout(() => {
    throw new Error("Error!");
  }, 100);
} catch (e) {
    // 에러 캐치X
  console.error("캐치한 에러", e);
}
```

### 답안 작성
```

```

<br>

## 📌문제3
### 다음 코드의 출력 순서를 작성하세요.
```js
setTimeout(() => console.log(1), 0); // (1)

Promise.resolve()
  .then(() => console.log(2)) // (2)
  .then(() => console.log(3)); // (3)
```

### 답안 작성
```

```

<br>

